recipes/hpv-kite/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Zatrzymaj przy błędzie
+set -o errexit
+
+# Budowanie projektu za pomocą Gradle
+sh ./gradlew clean build jar createDependenciesJar
+
+# Tworzenie docelowego katalogu dla Conda
+mkdir -p $PREFIX/bin
+
+# Kopiowanie pliku JAR
+cp build/libs/*.jar $PREFIX/bin/
```

Comment:
```suggestion
mkdir -p ${PREFIX}/libexec/hpv-kite
cp build/libs/*.jar ${PREFIX}/libexec/hpv-kite
```
Putting `.jar` files in `${PREFIX}/bin` can cause weird issues later for users.  A better approach is to use `${PREFIX}/libexec/hpv-kite`.